src/main/java/com/extremelyd1/game/Game.java

```diff
@@ -530,6 +543,39 @@ public void onMaterialCollected(Player player, Material material) {
                 // for other teams
             }
 
+            if (winConditionChecker.isFullCard()) {
+                if (config.getProgressController().shouldNotifyCountAmount(collectorTeam.getNumCollected())) {
+                    Bukkit.broadcastMessage(
+                            PREFIX +
+                                    collectorTeam.getColor() + collectorTeam.getName()
+                                    + ChatColor.WHITE + " team has collected "
+                                    + ChatColor.AQUA + collectorTeam.getNumCollected() + ChatColor.WHITE + " items"
+                    );
+                }
+            } else if (winConditionChecker.getNumLinesToComplete() > 0) {
+                int linesCompleteNow = bingoCard.getNumLinesComplete(collectorTeam);
+
+                if (linesCompleteNow > linesPreCheck) {
+                    if (config.getProgressController().shouldNotifyProgressLines(linesCompleteNow)) {
+                        Bukkit.broadcastMessage(
+                                PREFIX +
+                                        collectorTeam.getColor() + collectorTeam.getName()
+                                        + ChatColor.WHITE + " team has completed "
+                                        + ChatColor.AQUA + linesCompleteNow + ChatColor.WHITE + " lines"
+                        );
+                    }
+                }
+            } else if (winConditionChecker.getCompletionsToLock() > 0) {
+                if (config.getProgressController().shouldNotifyCountAmount(collectorTeam.getNumCollected())) {
```

Comment:
Shouldn't this use a different method in `ProgressController`?
```suggestion
                if (config.getProgressController().shouldNotifyLockoutAmount(collectorTeam.getNumCollected())) {
```

Reply:
this has been moved into the controller and change has been made.